testplan/testing/py_test.py

```diff
@@ -546,7 +548,7 @@ def pytest_exception_interact(self, node, call, report):
             )
             details = (
                 "File: {}\nLine: {}\n{}: {}".format(
-                    trace.path.strpath,
+                    str(trace.path),
```

Comment:
Breaking change from pytest version 7.0.0 that LocalPath objects are now replaced with Pathlike. In any case, the strpath attribute was not part of the "public" pytest interface at any point so this is a desired change in my book even if fixing pytest version at less than 7.